app/lib/katello/resources/candlepin/consumer.rb

```diff
@@ -214,6 +214,12 @@ def path(id = upstream_consumer_id)
             super(id)
           end
 
+          def remove_entitlement(entitlement_id)
+            fail ArgumentError, "No entitlement ID given to remove." if entitlement_id.blank?
+
+            self["entitlements/#{entitlement_id}"].delete
```

Comment:
Working well, other than the individual `RemoveEntitlement` output in Dynflow. You need to return a string to the action. Wrapping this in a JSON.parse would do the trick.

Reply:
Hmm... nevermind. Candlepin doesn't return anything here on success! Bummer!